### ISSUE TITLE
nomos false positive: trademark reference in PostScript files

### ISSUE THREAD

**Issue by armijnhemel** [fossology/fossology#814]
In PostScript files that embed fonts that have a definition for the trademark symbol Nomos will say that there is a trademark reference. An example is tech_report.ps in rsync 0.6:

    % 0x90
     /.notdef /.notdef /.notdef /quotedblleft
     /quotedblright /bullet /endash /emdash
     /tilde /trademark /scaron /guilsinglright
     /oe /.notdef /.notdef /Ydieresis

**Comment by mcjaeger** [comment]
I agree that the result "Trademark-Ref" should be moved out of nomos because it is not a license (result).

Would be a resolution to pull out licenses with the rf_active column for nomos just like monk?

https://github.com/fossology/fossology/pull/835

-> then users could just omit license results from Fossology.

**Comment by armijnhemel** [comment]
My point was that Nomos should omit scanning embedded fonts in Postscript files :-)

I don't really care how it is done, as long as Postscript files don't turn up in my Nomos results I'm fine with it.

**Comment by shaheemazmalmmd** [comment]
We have a feature to skip scanning of specific type of files using mimetype. the reference to this type of files can be added to admin > customize > Skip MimeTypes from scanning to skip the scanning.
